### ISSUE TITLE
train.py

### ISSUE THREAD

**Issue by HirataYurina** [DequanZhu/FaceNet-and-FaceLoss-collections-tensorflow2.0#4]
你好，请问在训练的时候 gradients = tape.gradient(loss, self.model.trainable_variables)，为什么反向传播的时候不需要计算最后的ArcFaceLoss中的weights呢？

**Comment by DequanZhu** [comment]
对，这里应该错了，你可以把loss_layer定义在facenet模型里
